### DISCUSSION TITLE
Suggestion for Phase 80 - All tiers and especially end of Phase 80 (I know it is a long shot)

### DISCUSSION THREAD

**Discussion by DPComputer** [chromiecraft/chromiecraft#4465] [Suggestions]
Hello friends.

**Bit of personal background (you can skip that if you want to):**

> I decided to stop giving Blizzard money when MoP was out and Siege of Orgrimmar was just released, reason being i didn't liked the direction the game was taking, so i've dedicated myself to playing WoTLK on private server since then.
> Have seen many variations, some silly, some extreme, some pretty close to the retail experience.
> Most of them if not all suffered from an intrinsic problem all PVT server suffers, which is lack of continuity.
> WoW is supposed to be progressing expansion after expansion.
> It's like a really bad drug, you get hooked, you can't stop and find yourself wanting even more and more with time.

### **The problems:**
- Once ICC is out, every other content prior to that becomes highly irrelevant and abandoned, which is a shame (Ulduar best raid.)
- At that point you find yourself doing PUGs with maybe fresh 80's with green gear hoping to skip the gear grind in order to do ICC in a "normal" way.
- Which leads to world PUGs having to ask for GS to do ICC which is never a good measure of capability.

**In my travels on different servers i've found but one that had a solution, not a perfect one, but close to it, for all those problems above, which is the subject of my suggestion today:**

### Introducing "**Raid Tier Soft Progression System**"

How does it work:

- It is based on per character achievements, and it goes as follows: In order to enter Ruby Sanctum you need the achievement "Fall of The Lich King", in order to enter ICC you need the achievement "Trial of the Crusader", in order to enter ToC you need "The secrets of Ulduar" and finally in order to enter Ulduar you need "The fall of Naxxramas"
- It Is difficulty based, aka 10 man achievements unlock 10 man raids, 25 man achievements unlock 25 man raids
- Emblems: Bosses will always drop that raid's correspondent emblems for everyone. Example: Players get [Emblems of Conquest](http://www.wowhead.com/item=45624/emblem-of-conquest) upon killing any boss inside Ulduar, no matter what is the character's progression.
- Vault of Archavon needs a bit of reworkign for that, due to item level drops and would go as follow: Archavon will be always available, regardless of the players progression as it was released with Naxxramas, thus since the very beginning of the WoTLK. However, Emalon was released as the same time as Ulduar, which implies that he will be available if each and every member has completed the Naxxramas raid. What if there are several players who have not completed it yet? The other bosses are not available for the raid until these players leave the instance, there will be just Archavon. If just one player does not meet the requirements, the bosses will disappear and the instance will configure itself in real time.

Some players may find it annoying having to do it for alts, but at the same time you not only solve the problems i mentioned above, but at the same time the world chat will be populated with PUGs for any raids.
People will do "Attunement Runs" making old content still viable and fun to play.

IMHO It will extend the lifetime of the server, once Phase 80 is out and wild.

I hope you take it into consideration.

With Love.

Icewallow

**Comment by balleny** [comment]
sounds interesting, but also has some drawbacks.
first of all, it is somehow similar with TBC content requiring attunements. Also, with release of Sunwell, all non t6 raids became kinda obsolete, similar to ICC for Wotlk.
In both cases, requirements to enter were removed (2.4. removed essentially all attunements to raids for TBC).

What you are describing is obviously the assumption that CC gonna stay more than 3-6 months on T10 content.
At the same time, you could consider saying that reaching that state, CC fulfilled is original intention of having a walkthrough the content to collect bug reports, and could be shut down. Then, the issue you are describing of people only doing t10 content wouldnt hold true fully.

My biggest concern would be that requiring people to complete all the dungeons beforehand, would likely require people to have a guild to get "attunement" runs organized. I would assume that the chances of getting into an ICC pug on Chromiecraft, with a bit lower gearscore, are more likely than full scale attunement runs through the whole content, as e.g. a casual or guildless person.

I could understand a system which requires you to clear the previous content first (which gradually shifts with each new release, hence you would have to do only 1 attunement raid). Still, not convinced.

**Reply by DPComputer** [reply]
> My biggest concern would be that requiring people to complete all the dungeons beforehand, would likely require people to have a guild to get "attunement" runs organized. I would assume that the chances of getting into an ICC pug on Chromiecraft, with a bit lower gearscore, are more likely than full scale attunement runs through the whole content, as e.g. a casual or guildless person.

During week, where there are no ICC runs, do you think noone will be bored enough to throw up a NAXX attune run?

I've seen this method live and it was like i described.

**Reply by balleny** [reply]
Oh on the contrary: I would assume that when ICC is out for a bit, there would be more ICC during weekdays. And only on the weekend, there would be enough "casual" online simultaneously for a potential pug attunement run (not guild)

**Comment by Santaria94** [comment]
Trying to force people to go to dead raids in order to get into actual raids will end up with players refusing to create new alts, and new players will leave the server without finding a group in order to attune. As a result, this will lead to a decrease in online and already problems with the organization of actual raids, and then the death of the server. Nice suggestion.

**Reply by DPComputer** [reply]
You underestimate the will of people wanting to play a good wotlk server

**Comment by Amrx2** [comment]
Hi DPComputer, this isn't something that we have in mind implementing on Chromiecraft at this moment in time. In theory, with a huge server it is doable, our concern would be gating people from current raids if they cannot find spots in older content long periods of times after it was released.

Once our progression is complete and we might look at different plans in the future, maybe! 

Thanks for the suggestion, have a good day!
Construct
